.github/workflows/release.yml

```diff
@@ -48,10 +47,11 @@ jobs:
       - name: Test CAE files
         run: npm run test:cae:files
 
-      - name: Publish to NPM
-        run: npm publish --access public
+      - name: Release
+        run: npx semantic-release
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
Wird dieses selbst erstellte Token für die Authentifizierung verwendet? Dieses wird von GitHub eigentlich automatisch als Secret für jeden Workflow angelegt:

"GitHub provides a token that you can use to authenticate on behalf of GitHub Actions, it automatically creates a GITHUB_TOKEN secret to use in your workflow.
The permissions of GITHUB_TOKEN are limited to the repository that contains your workflow."

Reply:
Ja, dazu habe ich verschiedene Angaben gelesen im Zusammenhang mit semantic-release(ev. zu wenig Rechte). Ich habe auch einen Fork von diesem Repo (https://github.com/joerdis101/red-sourcepoint-cmp/actions), der tatsächlich auch mit dem automatisch erstellten funktioniert. Wir können es also zunächst gern mit diesem automatischen probieren.